modules/tensor_mechanics/src/materials/ComputeLinearElasticPFFractureStress.C

```diff
@@ -49,54 +65,63 @@ ComputeLinearElasticPFFractureStress::computeQpStress()
   if (c > 1.0)
     cfactor = 0.0;
 
-  // Compute eigenvectors and eigenvalues of ustress
   RankTwoTensor eigvec;
   std::vector<Real> eigval(LIBMESH_DIM);
```

Comment:
old code or not please make this a member variable

Reply:
We could use standard vector here since we will create the ranktwotensor with fillfrominputvector function.